_pages/meetings.md

```diff
@@ -13,11 +13,12 @@ Telecons are held every other week on Fridays at 10:00 AM Mountain time. Present
 Meetings and telecon times are available as a [Google calendar](https://calendar.google.com/calendar?cid=NG42Z3YyaWZncDZyZ25rOGF1N2pzZjF1azBAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ).
 
 ### Community Meetings
+2019 Fall Python in Heliophysics Community Meeting, November 4-6, 2019 (at LASP).
 
-2019 Python in Heliophysics Community Meeting, May 21-23, 2019. 
+2019 Spring Python in Heliophysics Community Meeting, May 21-23, 2019 (at LASP). 
 * [Meeting materials](https://drive.google.com/drive/u/0/folders/171Ba3Mq3MIaEXoc9X91gZhaXHVjoJde2)
 
-2018 Python in Heliophysics Working Group Meeting, November 13-15, 2018. 
+2018 Python in Heliophysics Working Group Meeting, November 13-15, 2018 (at LASP). 
```

Comment:
The google calendar should be updated to include this information as well.

Reply:
I would also suggest that this PR include a blog post about this new meeting.

Reply:
Let's put that in another PR so that the mailing list info can go up right away. Julie's on it

Reply:
Will add to google calendar and create a blog in another PR